backend/pkg/logger/errors.go

```diff
@@ -0,0 +1,32 @@
+package logger
+
+import (
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+type ErrLoggerNotFound struct {
+	name abstraction.LoggerName
+}
+
+func (e *ErrLoggerNotFound) Error() string {
+	return "Logger not found: " + string(e.name)
+}
+
+type ErrCreatingFile struct {
```

Comment:
I see you use this to wrap another error, if you want to do that include it with the error and define an `Unwrap() error` method that returns the inner error. This is because the [errors](https://pkg.go.dev/errors#section-documentation) package works with this